README.md

```diff
@@ -65,27 +26,23 @@ After you're finished please remove all the comments and instructions!
 
 # 📖 [your_project_name] <a name="about-project"></a>
```

Comment:
- [ ] Please add the name of the project here instead of `your_project_name` to make your readme file more professional.